security/compliance_operator/compliance-operator-manage.adoc

```diff
@@ -13,7 +13,7 @@ include::modules/compliance-imagestreams.adoc[leveloffset=+1]
 
 include::modules/compliance-profilebundle.adoc[leveloffset=+1]
 
-[id="additional-resources_managing-the-compliance-operator"]
```

Comment:
I thought the way this was is how we are supposed to do this?
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies
(of course, searching in the repo I see all sorts of formats...)